react/src/base/inputs/DatePicker.stories.js

```diff
@@ -85,8 +75,59 @@ export const disabledDatePicker = () => (
 disabledDatePicker.story = {
   name: 'Disabled',
   parameters: {
-    jest: [
-      'SprkDatePickerInput',
-    ]
+    jest: ['SprkDatePickerInput'],
+  },
+};
+
+export const hugeDatePicker = () => (
```

Comment:
We added a huge datepicker?

Reply:
React Date picker huge was added as a story because we had a support issue that came up and someone didn't know how to do it. Or thought it wasn't available because it's technically a different component and we didn't have docs for it. So we decided to add it.

Reply:
@guern1kn ah okay, cool. Is it in the release notes that we added it?

Reply:
Does it need to be? We didn't really add any functionality. Just stories.